### DISCUSSION TITLE
Problem with toggle button

### DISCUSSION THREAD

**Discussion by MateLoncar** [neuefische/questions#588] [Data Analytics] [answered]
First I made a new marker for each click, but now I decided to make a button that activates/deactivates the creation of markers per click,but the button doesn't work for me even though I think I added everything correctly.[https://github.com/MateLoncar/capstone-project/commits/US5](url)

**Comment by moonwave99** [comment, marked as answer]
The solution is to pass the `isAddingMarker` state to the Map element so it can decide whether the markers should be appended or not.

Also the event propagation must be stopped in the PopUp in order to prevent the creation of a new marker when removing an old one.
